src/js/drop.js

```diff
@@ -246,7 +246,11 @@ function createContext(options={}) {
         targetOffset: '0 0',
         enabled: false,
         constraints: constraints,
-        addTargetClasses: this.options.addTargetClasses
+        addTargetClasses: this.options.addTargetClasses,
+        focusDelay: 0,
```

Comment:
Everything looks good, but I think these delays should go [here](https://github.com/caseyWebb/drop/blob/hover-delay/src/js/drop.js#L74). These options are for `Tether` specifically. Understandably, it's a bit confusing since `Drop` also accept similar, if not the same options :sweat_smile: